pkg/tasks/update_template_content.go

```diff
@@ -508,6 +528,75 @@ func (t *UpdateTemplateContent) getContentList() ([]caliri.ContentDTO, []string,
 	return contentToCreate, customContentIDs, rhContentIDs, nil
 }
 
+// getOverrideDTOs uses the RepoConfigUUIDs to query the db and generate a mapping of content labels to distribution URLs
+// for the snapshot within the template.  For all repos, we include an override for an 'empty' sslcacert, so it does not use the configured default
+// on the client.  For custom repos, we override the base URL, due to the fact that we use different domains for RH and custom repos.
+func (t *UpdateTemplateContent) getOverrideDTOs(contentPath string) ([]caliri.ContentOverrideDTO, error) {
+	mapping := []caliri.ContentOverrideDTO{}
+	uuids := strings.Join(t.payload.RepoConfigUUIDs, ",")
+	origins := strings.Join([]string{config.OriginExternal, config.OriginRedHat}, ",")
+	customRepos, _, err := t.daoReg.RepositoryConfig.List(t.ctx, t.orgId, api.PaginationData{Limit: -1}, api.FilterData{UUID: uuids, Origin: origins})
```

Comment:
this is redhat and custom, not just custom, right?

Reply:
correct, we have to override the sslcacert for both